requirements/test.txt

```diff
@@ -1,6 +1,7 @@
 # This file is used by Jenkins for testing,
 # it is not needed for local testing.
 -r prod.txt
+--no-binary Pillow
```

Comment:
We need to talk about the correct solution for this. I'm just adding this for now so that tests will run correctly.